repo_manager/utils/__init__.py

```diff
@@ -90,6 +90,13 @@ def validate_inputs(parsed_inputs: dict[str, Any]) -> dict[str, Any]:
 
     actions_toolkit.debug(f"api_url: {api_url}")
 
+    try:
```

Comment:
Why does this need to be duplicated? it seems to duplicate the lines directly below it